InsertData/testsrc/InsertDataTest.java

```diff
@@ -0,0 +1,190 @@
+		  
+		String[] orderfiles = new String[3];
+		
+		for (int i=0; i< orderfiles.length; i++)
+		{
+			orderfiles[i] = "order"+(i+1)+".json";
+		}
+		
+		List<Document> filteredDocumentsBefore = insertData.CountOrderSize(CONNECTION);
+		List<Document> AddDocuments = insertData.AddMultipleOrdersWithFiles(CONNECTION, orderfiles);
+		List<Document> filteredDocumentsAfter = insertData.CountOrderSize(CONNECTION);
+		
+		// check if the data is added and the overall size has incremented by the size of the array
+		assertEquals(filteredDocumentsBefore.size()+orderfiles.length, filteredDocumentsAfter.size());	
+		Document result = AddDocuments.get(filteredDocumentsAfter.size()-1);
+		String id = result.getString("_id");
+		assertEquals("3a333e3a3a33cc333fb3f33c",id);	
+	
+		
+	}
+	
+	@Test
+	public void AddMultipleOrderWithDataTest() {
+		
+		String[] sku = {"MAERW111","MBERQ112","MCERT113"};
+		String[] item = {"color pencil","marker","compass"};
+		Double[] unit = {5.0, 24.5, 35.20};
+		int[] quant= {7,14,23};
+		
+		InsertData insertData = new InsertData();		
+		List<Document> filteredDocumentsBefore = insertData.CountOrderSize(CONNECTION);
+		InsertData.AddMultipleOrderWithData(CONNECTION, sku, item, unit, quant);
+		List<Document> filteredDocumentsAfter = insertData.CountOrderSize(CONNECTION);
+			
+		assertEquals(filteredDocumentsBefore.size()+3, filteredDocumentsAfter.size());	
+	}
+
+	
+	@Test(expected=IllegalArgumentException.class)
+	public void AddMultipleOrderWithDataTest_InvalidConnectionString_ThrowsException() 
+	{
+		String[] sku = {"MAERW111","MBERQ112","MCERT113"};
+		String[] item = {"color pencil","marker","compass"};
+		Double[] unit = {5.0, 24.5, 35.20};
+		int[] quant= {7, 14,23};
+		
+		InsertData insertData = new InsertData();	
+		List<Document> AddDocuments = insertData.AddMultipleOrderWithData("",  sku, item, unit, quant );
+	}
+	
+	@Test(expected=IllegalArgumentException.class)
+	public void AddMultipleOrderWithDataTest_emptyValue_ThrowsException() 
+	{
+		String[] sku = {};
+		String[] item = {"color pencil","marker","compass"};
+		Double[] unit = {};
+		int[] quant= {7, 14,23};
+		
+		InsertData insertData = new InsertData();	
+		List<Document> AddDocuments = insertData.AddMultipleOrderWithData("",  sku, item, unit, quant );
+	}
+
+	
+	@Test
+	public void RemoveAddedOrders() 
+	{
+		
+		MongoClientURI clientUri = new MongoClientURI(CONNECTION);
+		MongoClient client = new MongoClient(clientUri);
+		MongoDatabase database = client.getDatabase("stores");
+		MongoCollection<Document> collection = database.getCollection("orders");
+		SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
+		Date date = new Date();
+		String today = formatter.format(date);
+		String[] ddatess = {today,"2018-03-14","2018-03-11", "2018-04-11"};
+		DeleteResult deletedOrder = collection.deleteMany(in("orderPlaced",ddatess));
+		
+	
+		System.out.println("Number of Orders removed: "+deletedOrder.getDeletedCount());
```

Comment:
I do not see any assert statement in this unit tests. Is something missing?

Reply:
I have made changes in the test case to remove the added data and restore to original state